duckduckgo/iOS
# Update autoconsent script to 4.3.3

Task/Issue URL: https://app.asana.com/0/414235014887631/1204879897439296/f

**Description**:
Update autoconsent script to latest version.

**Steps to test this PR**:
Smoke test automatic cookie popup management.
1. Ensure that "Manage cookie pop-ups" option in settings is enabled.
2. Use fire button to clear cookies.
3. Navigate to a website known to have cookie prompt e.g. "onet.pl" or "olx.pl" (accessed from EU IP).
4. Whens website completes loading after blocked tracker animation a "Cookise Managed" notification should be shown in the address bar.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
